physics/PBL/SATMEDMF/satmedmfvdifq.meta

```diff
@@ -655,12 +639,11 @@
 [cpopu]
   standard_name = canopy_population_density
   long_name = population density used for canopy correction
-  units = 10000people 10km-2
+  units = people km-2
```

Comment:
@mkavulich @grantfirl Are we ok with these units? People is just a counter, should it have no units (dimensionless)?

Reply:
Good question. I'm OK with this, although alternatives could be `count km-2` or just `km-2`. For conversion purposes, if this were to ever be converted using the CCPP framework, I can see that we should probably prefer `km-2`. @mkavulich Do you have any preference?

Reply:
@grantfirl Thank you for the clarification.  We will revise to use km-2 units here, in FV3 and UWM for this variable.